agent/fw_hooks.h

```diff
@@ -27,6 +27,7 @@ extern void nr_drupal_enable(TSRMLS_D);
 extern void nr_drupal8_enable(TSRMLS_D);
 extern void nr_joomla_enable(TSRMLS_D);
 extern void nr_kohana_enable(TSRMLS_D);
+extern void nr_fw_laminas3_enable(TSRMLS_D);
```

Comment:
Could we change the name here for consistency. `nr_fw_laminas3_enable` -> `nr_laminas3_enable`